XamlControlsGallery/ControlPages/TabViewPage.xaml.cs

```diff
@@ -26,10 +28,37 @@ public sealed partial class TabViewPage : Page
         public TabViewPage()
         {
             this.InitializeComponent();
+            Loaded += TabViewPage_Loaded;
 
             InitializeDataBindingSampleData();
         }
 
+        private void TabViewPage_Loaded(object sender, RoutedEventArgs e)
```

Comment:
This is a heavy hammer; yes, it avoids the crash, but this chunk of code is divorced from the problem sample. I'm following up over on the main thread.

If the resulting guidance is "disable the animation for the TabView", I think we'll want to localize the code to disable to the specific sample itself so it is more obvious what is happening and also to help increase copy-paste'ability.